src/shared/models/currencies.model.ts

```diff
@@ -1,5 +1,5 @@
-export type CryptoCurrencies = 'BTC' | 'STX';
+export type CryptoCurrencies = 'BTC' | 'STX' | string;
```

Comment:
Aha, we do already have this type. Maybe we can just use that where I mentioned above?

Reply:
https://github.com/sindresorhus/type-fest/blob/main/source/literal-union.d.ts